context/brand-context/springernature/scss/40-base/links.scss

```diff
@@ -1,20 +1,3 @@
-/**
- * Links
- * Basic link styles
- */
-
 a {
-	text-decoration: none;
-	vertical-align: baseline;
-	color: color('blue');
-}
-
-a:hover {
-	text-decoration: underline;
-	-webkit-text-decoration-skip: skip;
-	text-decoration-skip: ink;
-}
-
-a:active {
-	outline: 0;
+	@include base-link;
```

Comment:
make sure all rules (cheeck valign) are included

Reply:
Let's tackle this issue on another ticket. 
We can include the text-decoration-skip and check if we really need the vertical-align.
Then we can move the `base-link` mixin in the global context

Reply:
We will federate the `springer` and `springernature` another time, as this demand bigger refactor, bringing context variables to the global level first...